examples/demo-app/src/utils/cloud-providers/carto.js

```diff
@@ -71,7 +71,7 @@ async function uploadFile({blob, name: fileName, isPublic = true}) {
   }, cartoDatasets, true);
 
   return ({
-    url: `demo/map/carto/${result.id}?owner=${cs.username}`
+    url: `demo/map/carto?mapId=${result.id}&owner=${cs.getSQLClient().username}`
```

Comment:
Here we can access `username` through `carto`.
```suggestion
    url: `demo/map/carto?mapId=${result.id}&owner=${carto.username}`
```